{
  "uid" : "f28e644e421d3e5c",
  "name" : "Attempt to revoke an already revoked certificate -- @1.1 ",
  "fullName" : "Certificate lifecycle management via TPC_Web: Attempt to revoke an already revoked certificate",
  "historyId" : "4b4108fb6f1e9e2257944b6011187ab3",
  "time" : {
    "start" : 1746708241261,
    "stop" : 1746708241832,
    "duration" : 571
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Step not implemented: Set certificate to revoked state.\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 39, in step_certificate_revoked\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Step not implemented: Set certificate to revoked state.\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 39, in step_certificate_revoked\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746708241263,
        "stop" : 1746708241803,
        "duration" : 540
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Given the certificate CERT6161 exists and is revoked",
      "time" : {
        "start" : 1746708241803,
        "stop" : 1746708241803,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Step not implemented: Set certificate to revoked state.\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_003_steps.py\", line 39, in step_certificate_revoked\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "When the admin attempts to revoke the certificate CERT6161",
      "time" : {
        "start" : 1746708241832,
        "stop" : 1746708241832,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Then the system should display an error message stating \"Certificate is already revoked\"",
      "time" : {
        "start" : 1746708241832,
        "stop" : 1746708241832,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Certificate lifecycle management via TPC_Web"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "certificate_id",
    "value" : "CERT6161"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 3,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 3
      },
      "items" : [ {
        "uid" : "9ad109e3dd9302e2",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/17/#testresult/9ad109e3dd9302e2",
        "status" : "failed",
        "statusDetails" : "AssertionError: Step not implemented: Set certificate to revoked state.\n",
        "time" : {
          "start" : 1746705352839,
          "stop" : 1746705353410,
          "duration" : 571
        }
      }, {
        "uid" : "772573fdc2044c46",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/16/#testresult/772573fdc2044c46",
        "status" : "failed",
        "statusDetails" : "AssertionError: Step not implemented: Set certificate to revoked state.\n",
        "time" : {
          "start" : 1746616903993,
          "stop" : 1746616904571,
          "duration" : 578
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "f28e644e421d3e5c.json",
  "parameterValues" : [ "CERT6161" ]
}